boards/OMote/Makefile.dep

```diff
@@ -0,0 +1,8 @@
+ifneq (,$(filter gnrc_netdev_default netdev_default,$(USEMODULE)))
+  USEMODULE += cc2538_rf
+endif
+
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+  USEMODULE += si7006
```

Comment:
You didn't list this in your board feature list - is that sensor really on the board or is it a copy & paste artifact?